utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -178,19 +178,39 @@ def _cleanup_by_path(self, path):
 
 
 class OscapScan(object):
-    def __init__(self, tmp_dir=tempfile.gettempdir(), mnt_dir=os.getcwd(),
+    def __init__(self, tmp_dir=tempfile.gettempdir(), mnt_dir=None,
                  hours_old=2):
         self.tmp_dir = tmp_dir
         self.helper = OscapHelpers(tmp_dir)
         self.mnt_dir = mnt_dir
         self.hours_old = hours_old
 
+    def _get_mnt_dir(self):
+        '''
+        Get directory for mount container
+        '''
```

Comment:
According to its name and comment, this method looks like a getter, but actually it can create a directory. Could we have a better name or better comment?